app/models/classified_listing.rb

```diff
@@ -56,6 +57,22 @@ class ClassifiedListing < ApplicationRecord
 
   scope :published, -> { where(published: true) }
 
+  def index_to_elasticsearch
```

Comment:
Feel free to leave this as is, I just want to have conversation around it before we establish a pattern: my main concern here is that Rails models tend to become dumping grounds for many unrelated concepts, so I'm always reluctant to add to much more to them. Is there another place we could put these methods? The `Search` module already has `*Serializer`, maybe it could also have `*Indexer` or something? Just thinking out loud here.

Reply:
I feel that. It is easy to default to the model. In the past when using Elasticsearch I've also put it in the model, especially since this called in an `after_commit` callback.

I'll default to @mstruve on what she'd like to do in terms of establishing a pattern.

Reply:
Sure, no worries, I'll accept whatever we agree on as a team, just wanted to have a discussion around it *before* we accidentally establish a pattern.

The callback can always be changed to something like

```ruby
after_commit { Search::ClassifiedListintIndexer.something(id) }
```

Reply:
Ah, I think your edit overlapped with my comment again 😄

Reply:
Haha I think you're right! And yep, I agree about discussing it before it becomes a pattern.

Reply:
@citizen428 one step ahead of you! https://github.com/thepracticaldev/dev.to/pull/6160

I like the simplicity of having the methods though. As someone who has often had to get into a console and poke around and look at data, having those helper methods is HUGE!

Reply:
Fair point, thanks @mstruve!